PhysicsTools/PatAlgos/plugins/PATElectronSlimmer.cc

```diff
@@ -20,6 +20,7 @@
 #include "CommonTools/UtilAlgos/interface/StringCutObjectSelector.h"
 #include "RecoEcal/EgammaCoreTools/interface/EcalClusterLazyTools.h"
 #include "FWCore/Utilities/interface/isFinite.h"
+#include "DataFormats/Common/interface/ValueMap.h"
 
```

Comment:
there are no useful changes in this file anymore.
Better remove all edits, better yet ```git rebase -i ``` and remove commits that modified this file.